tests/unit/db/bigquery/test_bigquery_loader.py

```diff
@@ -78,21 +79,20 @@ def test_load(self, mock_load_job_config):
         mock_target_records_format.format_type = 'delimited'
         mock_target_records_format.hints = {}
         mock_directory = Mock(name='mock_directory')
+        mock_directory.scheme = 'gs'
         mock_url = Mock(name='mock_url')
         mock_directory.manifest_entry_urls.return_value = [mock_url]
 
         mock_connection = mock_db.engine.raw_connection.return_value.connection
         mock_client = mock_connection._client
-        mock_job = mock_client.load_table_from_file.return_value
+        mock_job = mock_client.load_table_from_uri.return_value
         mock_job.output_rows = 42
-        mock_loc = mock_file_url.return_value
-        mock_f = mock_loc.open.return_value.__enter__.return_value
         out = big_query_loader.load(schema=mock_schema, table=mock_table,
                                     load_plan=mock_load_plan,
                                     directory=mock_directory)
-        mock_file_url.assert_called_with(mock_url)
-        mock_client.load_table_from_file.\
-            assert_called_with(mock_f,
+        # mock_file_url.assert_called_with(mock_url) # TODO
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker